build-system/app-index/amphtml-helpers.js

```diff
@@ -33,15 +32,13 @@ const containsByRegex = (str, re) => str.search(re) > -1;
 // TODO(alanorozco): Expand
 const formTypes = ['input', 'select', 'form'];
 
-
 const ExtensionScript = ({name, version, isTemplate}) =>
   html`<script
     async
     ${isTemplate ? 'custom-template' : 'custom-element'}="${name}"
-    src="https://cdn.ampproject.org/v0/${name}-${version || '0.1'}.js">
+    src="${process.env.OFFLINE ? '/dist' : 'https://cdn.ampproject.org'}/v0/${name}-${version || '0.1'}${process.env.OFFLINE ? '.max' : ''}.js">
```

Comment:
Let's do this as a post-render step and use `replaceUrls` to get the same compiled mode as currently running:

https://github.com/ampproject/amphtml/blob/391a431816894cc726e8c7891081885589f391bf/build-system/app-utils.js#L24-L88

Reply:
Sounds good!